Dockerfile-ngc-hpc

```diff
@@ -38,6 +38,11 @@ RUN if [ -n "${WITH_NCCL}" ]; then ${SCRIPT_DIR}/build_nccl.sh; fi
 ENV NCCL_LIB_DIR=${HOROVOD_NCCL_HOME}/lib
 ENV LD_LIBRARY_PATH=${WITH_NCCL:+$NCCL_LIB_DIR:}$LD_LIBRARY_PATH
 
+# We run this here even though it might be a repeat from the base image
```

Comment:
If the base image doesn't have the packages needed to build cxi, should we move this section up 10 lines to insure the packages needed to build NCCL are present?